packages/@aws-cdk/aws-cognito/test/user-pool.test.ts

```diff
@@ -454,6 +454,23 @@ describe('User Pool', () => {
     });
   });
 
+  test('case sensitive is correctly picked up', () => {
```

Comment:
how about a test to verify the default of true is set?

Reply:
The default comes from CloudFormation, not from CDK, meaning that the template won't actually contain this parameter at all but case sensitivity will be enabled once the user pool is created. Since that logic isn't in CDK, is that something we should verify? Or should we not mention the default in this case?

Reply:
gotcha, then I think it's fine. I believe there's already a test that verifies the default settings right? 
That should cover the scenario where this property is unset.

feel free to resolve!

Reply:
Add a test case instead to verify that the value is `ABSENT` by default.